Cargo.toml

```
@@ -64,6 +64,7 @@ indexmap = "1.6.1"
 notify-rust = { version = "4.2.2", optional = true }
 semver = "0.11.0"
 which = "4.0.2"
+process_control = "3.0.0"
```

This features is supposed to speed the crate up and it's already an indirect dependency.

```suggestion
process_control = { version = "3.0.0", features = ["crossbeam-channel"] }
```

---

Good idea, I never thought to add while working on the poc.